pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/RexExpression.java

```diff
@@ -106,6 +113,33 @@ static FieldSpec.DataType toDataType(RelDataType type) {
     }
   }
 
+  // POC Code. Will clean-up later.
+  static RexExpression handleSearch(RexCall rexCall) {
```

Comment:
we can create a new util class call `RexExpressionUtils` to handle special SqlKinds